recipes/mdahole2/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "mdahole2" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1af9c3bb6f07f6266aeb038fef5a1fa63dafb70a466dcdfe44b6627e8d9aec59
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
This can probably be noarch.

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
Thanks for the review @ocefpaf -- changed it to noarch!